framework/decode/openxr_replay_consumer_base.cpp

```diff
@@ -836,43 +882,72 @@ void OpenXrReplayConsumerBase::Process_xrPollEvent(const ApiCallInfo&
             replay_result  = GetInstanceTable(in_instance)->PollEvent(in_instance, out_eventData);
             retry_count++;
 
+            GFXRECON_LOG_INFO("Looking for event %s (0x%x, %u)",
+                              GetEventTypeString(capture_event->type),
+                              capture_event->type,
+                              capture_event->type);
+
+            // If the event returned is not the one we expect, we may still want to keep track of
+            // it or bail.
```

Comment:
Shouldn't we check for SUCCESS before we inspect the out_eventData?

Reply:
Probably a good point.